fiftyone/migrations/revisions/v0_16_1.py

```diff
@@ -0,0 +1,296 @@
+"""
+FiftyOne v0.16.1 revision.
```

Comment:
Probably worth adding a comment there that explains what this migration is fixing. Otherwise it will look weird because, the way it is implemented, it seems to duplicate much of the v0.16.0's functionality.